test.html

```diff
@@ -1,7 +1,7 @@
 <html>
   <head>
     <style>
-      .chart-item-bg {stroke: #ddd}
+      .chart-item-bg {stroke: #000000}
```

Comment:
This was intentional. Switching the stroke to black made it much clearer to debug at the very very small levels that I was dealing with where it would break trying to cover off the closest acceptable fraction close to 1 / 0. I wanted the highest possible contrast.